api-extractor/src/ExternalApiHelper.ts

```diff
@@ -0,0 +1,49 @@
+import * as ts from 'typescript';
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import Analyzer from './Analyzer';
+import ApiJsonGenerator from './generators/ApiJsonGenerator';
+
+export default class ExternalApiHelper {
+
+  public static generateApiJson(rootDir: string, libFolder: string, externalPackageFilePath: string): void {
```

Comment:
Document these parameters.  Mainly it would be helpful to have an example string for each parameter (e.g. is it a relative or absolute path, etc)